### PR TITLE
SONAR-9424 Show onboarding tutorial on first login

### PR DESCRIPTION
**Pull request by stas-vilchik**

### CODE REVIEW

**Comment by SonarTech** [comment]
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [DefaultWsClient.java#L49](https://github.com/SonarSource/sonarqube/blob/4bfcbf0d6e79f4d372bebd6fa73b3804f37fe846/sonar-ws/src/main/java/org/sonarqube/ws/client/DefaultWsClient.java#L49): Refactor this class so it has no more than 20 fields, rather than the 21 it currently has. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1820)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Comment by SonarTech** [comment]
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [DefaultWsClient.java#L49](https://github.com/SonarSource/sonarqube/blob/796941341396808e2193808d5bd5e57cdddb9e49/sonar-ws/src/main/java/org/sonarqube/ws/client/DefaultWsClient.java#L49): Refactor this class so it has no more than 20 fields, rather than the 21 it currently has. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1820)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Comment by SonarTech** [comment]
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [DefaultWsClient.java#L49](https://github.com/SonarSource/sonarqube/blob/69bc513d74afae3b1acdcf3d964a002f4a459010/sonar-ws/src/main/java/org/sonarqube/ws/client/DefaultWsClient.java#L49): Refactor this class so it has no more than 20 fields, rather than the 21 it currently has. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1820)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Comment by SonarTech** [comment]
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [DefaultWsClient.java#L49](https://github.com/SonarSource/sonarqube/blob/f577a657469ec0d1da565efa23b1daba744d3dce/sonar-ws/src/main/java/org/sonarqube/ws/client/DefaultWsClient.java#L49): Refactor this class so it has no more than 20 fields, rather than the 21 it currently has. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1820)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Comment by SonarTech** [comment]
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [DefaultWsClient.java#L49](https://github.com/SonarSource/sonarqube/blob/d1e8ef54a9b0e1e4778518cff71e608a202a28f0/sonar-ws/src/main/java/org/sonarqube/ws/client/DefaultWsClient.java#L49): Refactor this class so it has no more than 20 fields, rather than the 21 it currently has. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1820)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Comment by SonarTech** [comment]
SonarQube analysis reported 1 issue

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [DefaultWsClient.java#L49](https://github.com/SonarSource/sonarqube/blob/193adce4cf9443f81a47ec2b9458b3bac35337b8/sonar-ws/src/main/java/org/sonarqube/ws/client/DefaultWsClient.java#L49): Refactor this class so it has no more than 20 fields, rather than the 21 it currently has. [rule](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1820)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

File: server/sonar-web/src/main/js/app/components/help/GlobalHelp.js [hunk 1]
```diff
@@ -28,7 +28,9 @@ import TutorialsHelp from './TutorialsHelp';
 import { translate } from '../../../helpers/l10n';
 
 type Props = {
+  currentUser: { isLoggedIn: boolean },
```
Comment (gregaubert) [inline server/sonar-web/src/main/js/app/components/help/GlobalHelp.js]: Is it passed there only to re-render the component when changing ?  I can't find any use of it. Maybe add a comment.

File: server/sonar-web/src/main/js/app/components/nav/global/GlobalNav.js [hunk 2]
```diff
@@ -57,8 +73,24 @@ class GlobalNav extends React.PureComponent {
 
   closeHelp = () => this.setState({ helpOpen: false });
 
+  openOnboardingTutorial = () => this.setState({ helpOpen: false, onboardingTutorialOpen: true });
+
+  closeOnboardingTutorial = () => this.setState({ onboardingTutorialOpen: false });
+
+  /* eslint-disable max-len */
+  renderHelpIcon = () => (
```
Comment (gregaubert) [inline server/sonar-web/src/main/js/app/components/nav/global/GlobalNav.js]: Would be nice to have the `HelpIcon` in `js/components/icon-components`

File: server/sonar-web/src/main/js/app/components/help/GlobalHelp.js (hunk 1 above)
Comment (stas-vilchik) [inline server/sonar-web/src/main/js/app/components/help/GlobalHelp.js]: Actually I forgot to use it to hide tutorials for anonymous users 🙀

**Pull request merged**